messages/en.json

```diff
@@ -1,10 +1,10 @@
 {
-  "admin/editor.image.title": "Image",
-  "admin/editor.image.description": "Show any image",
-  "admin/editor.image.src.title": "Image",
-  "admin/editor.image.alt.title": "Alternative text",
-  "admin/editor.blockClass.title": "CSS Block Class",
-  "admin/editor.blockClass.description": "Adds an extra class name to ease styling",
+  "admin/editor.store-image.title": "Image",
+  "admin/editor.store-image.description": "Show any image",
+  "admin/editor.store-image.src.title": "Image",
+  "admin/editor.store-image.alt.title": "Alternative text",
+  "admin/editor.store-image.blockClass.title": "CSS Block Class",
+  "admin/editor.store-image.blockClass.description": "Adds an extra class name to ease styling",
```

Comment:
wouldn't the pattern be `camel case`?

Reply:
Not in this case, because `store-image` is the name of the component I'm defining messages to, while the camel case is used for property names inside of my `schema`.

Reply:
Is this not inconsistent? It would be nice to keep the same pattern regardless of place.